src/types/codeHighlight.ts

```diff
@@ -29,16 +29,18 @@ export function isSupportedLanguage(lang: unknown): lang is SupportedLanguage {
 }
 
 export type UnformattedMessagePayload = {
+  id: string,
   text: string,
   lang: SupportedLanguage,
 }
 
 export type FormattedMessagePayload = {
-  unformatted: string,
+  id: string,
+  unformatted?: string | undefined,
   formatted: string,
   illegal: boolean,
   relevance: number,
-  language: string,
+  language?: string | undefined,
```

Comment:
These types were technically incorrect before but we're not using the return values anywhere in the component and the worker message wasn't typed